internal/raft/message/convert.go

```diff
@@ -0,0 +1,971 @@
+	msgListValues := &List_Values{}
+	msgListValues.Values.Expr = ConvertCommandToMessageRepeatedExprSlice(cmd.Values)
+	return msgListValues
+}
+
+// ConvertCommandToMessageList converts
+func ConvertCommandToMessageList(cmd command.List) *List {
+	msgList := &List{}
+	switch cmd := cmd.(type) {
+	case *command.Scan:
+		msgList.List = ConvertCommandToMessageListScan(cmd)
+	case *command.Select:
+		msgList.List = ConvertCommandToMessageListSelect(cmd)
+	case *command.Project:
+		msgList.List = ConvertCommandToMessageListProject(cmd)
+	case *command.Join:
+		msgList.List = ConvertCommandToMessageListJoin(cmd)
+	case *command.Limit:
+		msgList.List = ConvertCommandToMessageListLimit(cmd)
+	case *command.Offset:
+		msgList.List = ConvertCommandToMessageListOffset(cmd)
+	case *command.Distinct:
+		msgList.List = ConvertCommandToMessageListDistinct(cmd)
+	case *command.Values:
+		msgList.List = ConvertCommandToMessageListValues(cmd)
+	}
+	return msgList
+}
+
+// ConvertCommandToMessageSelect converts a Command type to a Command_Select type.
+func ConvertCommandToMessageSelect(cmd command.Command) *Command_Select {
+	msgCmdSelect := &Command_Select{}
+	msgCmdSelect.Filter = ConvertCommandToMessageExpr(cmd.(*command.Select).Filter)
+	msgCmdSelect.Input = ConvertCommandToMessageList(cmd.(*command.Select).Input)
+	return msgCmdSelect
+}
+
+// ConvertCommandToMessageCol converts command.Column to a message.Column
+func ConvertCommandToMessageCol(cmd command.Column) *Column {
+	msgCol := &Column{}
+	msgCol.Table = cmd.Table
+	msgCol.Column = ConvertCommandToMessageExpr(cmd.Column)
+	msgCol.Alias = cmd.Alias
+	return msgCol
+}
+
+// ConvertCommandToMessageCols converts []command.Column to a []message.Column
+func ConvertCommandToMessageCols(cmd []command.Column) []*Column {
+	msgCols := []*Column{}
+	for i := range cmd {
+		msgCols = append(msgCols, ConvertCommandToMessageCol(cmd[i]))
+	}
+	return msgCols
+}
+
+// ConvertCommandToMessageProject converts a Command type to a Command_Project type.
+func ConvertCommandToMessageProject(cmd command.Command) *Command_Project {
+	msgCmdProject := &Command_Project{}
+	msgCmdProject.Cols = ConvertCommandToMessageCols(cmd.(*command.Project).Cols)
+	msgCmdProject.Input = ConvertCommandToMessageList(cmd.(*command.Project).Input)
+	return msgCmdProject
+}
+
+// ConvertCommandToMessageDelete converts a Command type to a Command_Delete type.
+func ConvertCommandToMessageDelete(cmd command.Command) *Command_Delete {
+	msgCmdDelete := &Command_Delete{}
+	msgCmdDelete.Table = ConvertCommandToMessageTable(cmd.(*command.Delete).Table)
+	msgCmdDelete.Filter = ConvertCommandToMessageExpr(cmd.(*command.Delete).Filter)
+	return msgCmdDelete
+}
+
+// ConvertCommandToMessageDrop converts a Command type to a CommandDrop type.
+func ConvertCommandToMessageDrop(cmd command.Command) *CommandDrop {
+	msgCmdDrop := &CommandDrop{}
+	switch cmd := cmd.(type) {
+	case *command.DropTable:
+		msgCmdDrop.Target = 0
```

Comment:
Magic values?

Reply:
Actually referred from the enum. Maybe I should link to them sorry